aws/resource_aws_sagemaker_image_version_test.go

```diff
@@ -0,0 +1,190 @@
+package aws
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/sagemaker/finder"
+)
+
+func TestAccAWSSagemakerImageVersion_basic(t *testing.T) {
+
+	if os.Getenv("SAGEMAKER_IMAGE_VERSION_BAES_IMAGE") == "" {
```

Comment:
`SAGEMAKER_IMAGE_VERSION_BAES_IMAGE` -> `SAGEMAKER_IMAGE_VERSION_BASE_IMAGE` (spelling) 😄.
Also, add to the [_Maintaining the Terraform AWS Provider_ guide](https://github.com/hashicorp/terraform-provider-aws/blob/master/docs/MAINTAINING.md#environment-variable-dictionary).